wdio.conf.js

```diff
@@ -168,10 +168,8 @@ exports.config = {
         // <string[]> (name) specify the profile to use
         profile: [],
         // <string[]> (file/dir) require files before executing features
-        require: [
-            './node_modules/qustodio-itui/src/steps/given.js',
-            './node_modules/qustodio-itui/src/steps/when.js',
-            './node_modules/qustodio-itui/src/steps/then.js',
+        require: [ 
+          './src/steps/**/*.feature',
```

Comment:
This one I don't fully understand it. We're removing the steps (why?) and setting the path where we expect to find the features... That should be a responsibility of the project using `itui`, but not prescriptive from `itui` directly, no? Maybe there's some other intent here that I'm not seeing, just asking to uderstand here :)